src/pages/home-page/model/store.ts

```diff
@@ -0,0 +1,12 @@
+import { createEvent, createStore } from 'effector';
+
+export const setUsername = createEvent<string>();
+export const setPassword = createEvent<string>();
+
+export const $usernameStore = createStore('').on(setUsername, (_, value) => value);
+export const $passwordStore = createStore('').on(setPassword, (_, value) => value);
+
+export const handleSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
```

Comment:
this one shouldnt be here for sure. you can simply use the event on the UI that will trigger the effect in store.

for this reason, we are using an effector sample

so how it will work, you will have an event, let's say loginUserRequested event

then you're declaring a sample
`
sample({
  clock: loginUserReqested,  -> it means this sample will be triggered on event
  fn: () => 'data you want to pass to the function in case you need it',
  target: loginFx, -> this will be called
})
`